### DISCUSSION TITLE
Golang Merge Struct Like Javascript Object Spread Syntax

### DISCUSSION THREAD

**Discussion by giscus** [krehwell/P-Blog#10] [General]
# Golang Merge Struct Like Javascript Object Spread Syntax

How to merge struct in Golang just like how easy it is to merge an object in Javascript

https://krehwell.com/blog/golang-merge-struct-just-like-javascript-merge-object-with-spread-syntax

**Comment by nawafilhusnul** [comment]
implement go routine, so working with big struct will be great, I guess
